pkg/controller/common/license/fixtures_test.go

```diff
@@ -45,6 +45,8 @@ func asRuntimeObjects(l EnterpriseLicense, sig []byte) []runtime.Object {
 			ObjectMeta: metav1.ObjectMeta{
 				Namespace: "test-system",
 				Name:      "test-license",
+				// TODO (sabo): is this EnterpriseLicense.Type -> License.Type cast safe?
+				Labels: LabelsForType(LicenseType(l.License.Type)),
```

Comment:
ping @pebrc

Reply:
as discussed offline, they are two different things:
License.Type distinguishes `Elasticsearch` (i.e. cluster level) and `Enterprise` licenses
EnterpriseLicense.Type enumerates the two different kinds of Enterprise license we have: regular `enterprise` license and `enterprise-trial` license

Reply:
I think I'm a little bit confused still. I get that `LicenseType` is enterprise vs elasticsearch license, and the `EnterpriseLicenseType` is full enterprise vs enterprise trial. I think I might just need to go over it with you and then add some comments to the src in the future because the intent is hard for me to decipher at least